packages/wallets/src/features/cfd/screens/DynamicLeverage/DynamicLeverageMarketCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC } from 'react';
+import { useDynamicLeverage } from '@deriv/api';
+import { WalletText } from '../../../../components/Base';
+import { DynamicLeverageTableColumnHeader } from './DynamicLeverageTableColumnHeader';
+import './DynamicLeverageMarketCard.scss';
+
+type TDynamicLeverageMarketCardProps = NonNullable<
+    Exclude<ReturnType<typeof useDynamicLeverage>['data'][0], 'undefined'>
+>;
+
+export const DynamicLeverageMarketCard: FC<TDynamicLeverageMarketCardProps> = ({
+    data,
+    description,
+    leverage,
+    title,
+}) => (
+    <div className='wallets-dynamic-leverage-modal__market'>
+        <div className='wallets-dynamic-leverage-modal__market-title'>
+            <WalletText align='center' data-testid='market_title' size='sm' weight='bold'>
+                {title}
+            </WalletText>
+            {!!description && (
+                <WalletText align='center' data-testid='description_title' size='2xs' style={{ fontStyle: 'italic' }}>
```

Comment:
```suggestion
                <WalletText align='center' data-testid='description_title' size='2xs' style={{ fontStyle: 'italic' }}>
```
Best to extend the `WalletText` component in this case

Reply:
Updated.